### ISSUE TITLE
please tell me how to install peer dependencies (in README.md?)

### ISSUE THREAD

**Issue by archenemies** [jaebradley/flights-search-cli#75]
I'm trying to install the latest version, and I get stuck:

    $ sudo npm install flights-search-cli --global
    /usr/bin/flights -> /usr/lib/node_modules/flights-search-cli/build/executables/flights.js
    npm WARN @babel/plugin-transform-runtime@7.12.1 requires a peer of @babel/core@^7.0.0-0 but none is installed. You must install peer dependencies yourself.

    + flights-search-cli@1.1.10
    updated 1 package in 3.256s

How do I install a peer dependency? Thanks!

**Comment by jaebradley** [comment]
@archenemies thanks for opening up this issue.

Should be closed by #80 which was released as part of `v1.1.11` (https://github.com/jaebradley/flights-search-cli/releases/tag/v1.1.11).

When I install on my machine, here's what I see

```bash
npm i flights-search-cli -g

/Users/jaebradley/.nvm/versions/node/v12.14.1/bin/flights -> /Users/jaebradley/.nvm/versions/node/v12.14.1/lib/node_modules/flights-search-cli/build/executables/flights.js

> core-js@3.6.5 postinstall /Users/jaebradley/.nvm/versions/node/v12.14.1/lib/node_modules/flights-search-cli/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

+ flights-search-cli@1.1.11
added 79 packages from 58 contributors in 5.185s
```

If you can confirm that you are no longer seeing the reported error, feel free to close this issue.

**Comment by archenemies** [comment]
Thanks yes it seems to be fixed. Now I can travel again!
